packages/transition-frontend/src/components/forms/csv/widgets/FieldMappingWidgets.tsx

```
@@ -0,0 +1,170 @@
+ */
+export const GeographicProjectionWidget = (props: Omit<FieldMappingWidgetProps, 'csvFieldChoices' | 'required'>) => {
+    const { t } = useTranslation('main');
+    const projectionChoices = Object.values(Preferences.get('proj4Projections')).map((projection: any) => {
+        return {
+            value: projection.srid.toString(),
+            label: projection.label
+        };
+    });
+    return (
+        <div className="apptr__form-input-container _two-columns _small-inputs">
+            <label>{t(props.i18nlabel)}</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)